scripts/python/phoedf.py

```diff
@@ -80,11 +81,37 @@ def flatten_indices(mu, nu):
                 self.flatgcov[:, flatten_indices(1, 1), :, :, :] = 1.0
                 self.flatgcov[:, flatten_indices(2, 2), :, :, :] = 1.0
                 self.flatgcov[:, flatten_indices(3, 3), :, :, :] = 1.0
+            else:
+                assert (
+                    False
+                ), "Geometry not provided in dump file but also not Minkowski!"
         self.gcov = np.zeros([self.NumBlocks, 4, 4, self.Nx3, self.Nx2, self.Nx1])
+
+        # Account for included ghost zones in flattened geometry data
```

Comment:
With the new variable output logic in parthenon, gcov, which is technically not a proper cell-centered variable because it can suppress the X3 dimension in 3D, no longer had its ghost cells automatically removed along with the proper cell-centered variables when this is requested. So here we remove the ghost zones if necessary to match the rest of the data.